### PR TITLE
Use CLI to create flavors

### PR DESCRIPTION
**Pull request by evrardjp**
There is no good version (read: with compatible requirements)
for together shade, pbr, os client tools, and another conflicting
package.

This removes the dependency to shade and works out of the box.
Idempotency is ensured for existing/non existing flavors, without
checking their values.

This can be rolled back when the requirements will be bumped (i.e.
when we are in an upper branch, meant for the usage of shade and
these modules)

Connected https://github.com/rcbops/rpc-openstack/issues/2227

### CODE REVIEW

File: rpcd/playbooks/roles/rpc_support/tasks/create_default_flavors.yml [hunk 1]
```diff
@@ -13,33 +13,21 @@
...
-    ram: "{{ item.ram }}"
-    auth:
-      auth_url: "{{ keystone_service_internalurl }}"
-      username: "admin"
-      password: "{{ keystone_auth_admin_password }}"
-      project_name: "admin"
-      domain_name: "Default"
-    validate_certs: false
+  shell: |
+    openstack --os-cloud default flavor create --ram {{ item.RAM }} --disk {{ item.Disk }} --vcpus {{ item.VCPUs }} --public {{ item.Name }}
```
Comment (odyssey4me) [inline rpcd/playbooks/roles/rpc_support/tasks/create_default_flavors.yml]: I think you may need to source the openrc file, unless the clouds.yaml file is automatically discovered?

Comment (evrardjp) [inline rpcd/playbooks/roles/rpc_support/tasks/create_default_flavors.yml]: yeah clouds.yaml always auto discovered. I think it's the os_client lib that takes care of this.

**Comment by evrardjp** [comment]
Also, good point, I changed the dict variable names to match upstream column headers, if we ever need to fetch results and do a | difference.

**Comment by alextricity25** [comment]
@evrardjp Could we get the flavor CLI change backported to the newton-14.0 branch?

**Pull request merged**